src/pages/popup/index.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled, { createGlobalStyle } from 'styled-components';
+import Post from '../../components/Post/Post.js';
+
+const Container = styled.div`
+  padding: 16px;
+`;
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    width: 20rem;
+    margin:0;
+    font-color: #666666;
+  }
+`;
+
+
+
+/*
+
+.link-more {display: none;}
+
+.post.active .accordion, .accordion:hover {
+    background-color: #e7e7e8;
+}
+
+.post.active .accordion:after {
+  content: "-";
+  font-size: 42px;
+}
+
+*/
+
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+/**
+ * Initialize this view
+ */
+async function init() {
```

Comment:
Init is no longer being used, I have not replaced the remaining functionality within it yet.